recipes/llama-utils/meta.yaml

```diff
@@ -36,7 +36,6 @@ about:
   home: https://github.com/Serapieum-of-alex/llama-utils
   license: GPL-3.0-only
   license_family: GPL
-  license_file: LICENSE
```

Comment:
Just the file. You do need this so conda can find it in the tarball. Please act on the review comments only.